website/pages/docs/upgrading/upgrade-to-1.5.0.mdx

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.5.0 - Guides
+sidebar_title: Upgrade to 1.5.0
+description: |-
+  This page contains the list of deprecations and important or breaking changes
+  for Vault 1.5.0. Please read it carefully.
+---
+
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.4.x compared to 1.5.0. Please read it carefully.
+
+## Raft Configuration
+
+A new Raft configuration value, `max_entry_size`, has been introduced. This value
+limits the size in bytes for a Raft entry. It applies to both Put operations and
+transactions. Any put or transaction operation exceeding this configuration value
+will cause the respective operation to fail. The default value for this
+configuration is 1048576 -- two times the chunking size.
+
+In addition, a new metric has been introduced, `entrySize`, that allows for
+operators to view the average entry size and adjust the configuration value as
```

Comment:
I'm not entirely sure this is correct to the implementation but I think something along these lines better reflects what we talked about this morning.
```suggestion
operators to sample the entry size, view the average, and adjust the configuration value as
```